README.md

```diff
@@ -5,6 +5,17 @@ It is the source for the Spark package in universe.  If you wish to modify
 that package, you should do so here, and generate a new package as
 described below.
 
+Service documentation can be found here : [DC/OS Apache Spark Documentation](https://docs.mesosphere.com/services/spark/)
+
+
+## Integration Test Builds Matrix
+
+
+|  | DC/OS 1.12 | DC/OS 1.13 | DC/OS 2.0 | DC/OS Master |
+|------------|------------|------------|------------|------------|
+| Permissive | <a  href ="https://teamcity.mesosphere.io/viewType.html?buildTypeId=DataServices_Spark_Nightly_DCOS_112_SparkBuild_112_Permissive&guest=1" ><img  src ="https://teamcity.mesosphere.io/app/rest/builds/buildType:(id:DataServices_Spark_Nightly_DCOS_112_SparkBuild_112_Permissive)/statusIcon" /></a > | <a  href ="https://teamcity.mesosphere.io/viewType.html?buildTypeId=DataServices_Spark_Nightly_DCOS_113_SparkBuild_113_Permissive&guest=1" ><img  src ="https://teamcity.mesosphere.io/app/rest/builds/buildType:(id:DataServices_Spark_Nightly_DCOS_113_SparkBuild_113_Permissive)/statusIcon" /></a > | <a  href ="https://teamcity.mesosphere.io/viewType.html?buildTypeId=DataServices_Spark_Nightly_DCOS_20_SparkBuild_20_Permissive&guest=1" ><img  src ="https://teamcity.mesosphere.io/app/rest/builds/buildType:(id:DataServices_Spark_Nightly_DCOS_20_SparkBuild_20_Permissive)/statusIcon" /></a > | <a  href ="https://teamcity.mesosphere.io/viewType.html?buildTypeId=DataServices_Spark_Nightly_DCOS_master_SparkBuild_master_Permissive&guest=1" ><img  src ="https://teamcity.mesosphere.io/app/rest/builds/buildType:(id:DataServices_Spark_IntegrationTests_DCOS_master_Permissive)/statusIcon" /> </a > |
```

Comment:
For the master icon, I think the value is not referring to the correct ID. It should be Nightly Master build, so `DataServices_Spark_IntegrationTests_DCOS_master_Permissive` should be `DataServices_Spark_Nightly_DCOS_master_SparkBuild_master_Permissive`. WDYT?

Reply:
changed